clr-k8s-examples/setup_system.sh

```diff
@@ -7,7 +7,7 @@ ADD_NO_PROXY="10.244.0.0/16,10.96.0.0/12"
 ADD_NO_PROXY+=",$(hostname -I | sed 's/[[:space:]]/,/g')"
 
 #Install kubernetes and crio
-sudo -E swupd repair --picky -m 29880 --force
```

Comment:
Do you also disable swupd auto-update somewhere? Otherwise locking onto a specific version here just helps with the initial installation, later system updates then can still break the Kubernetes cluster (either because of bugs or because kubelet gets updated to a new major version before the admin has a chance to update the control plane).